docs/core/docker/build-container.md

```diff
@@ -22,30 +22,19 @@ You'll learn to:
 
 You'll understand the Docker container build and deploy tasks for a .NET Core application. The *Docker platform* uses the *Docker engine* to quickly build and package apps as *Docker images*. These images are written in the *Dockerfile* format to be deployed and run in a layered container.
 
+> [!TIP]
+> If you're working with an existing ASP.NET Core application, use the [Learn how to containerize an ASP.NET Core application.](/aspnet/core/host-and-deploy/docker/building-net-docker-images) tutorial.
+
 ## Prerequisites
 
 Install the following prerequisites:
 
-- [.NET Core 2.2 SDK](https://dotnet.microsoft.com/download)\
+- [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download)\
 If you have .NET Core installed, use the `dotnet --info` command to determine which SDK you're using.
 
 - [Docker Community Edition](https://www.docker.com/products/docker-desktop)
 
-- A temporary working folder for the *Dockerfile* and .NET Core example app. In this tutorial, the name `docker-working` is used as the working folder.
-
-### Use SDK version 2.2
-
-If you're using an SDK that is newer, like 3.0, make sure that your app is forced to use the 2.2 SDK. Create a file named *global.json* in your working folder and paste in the following JSON code:
-
-```json
-{
-  "sdk": {
-    "version": "2.2.100"
-  }
-}
-```
-
-Save this file. The presence of file will force .NET Core to use version 2.2 for any `dotnet` command called from this folder and below.
+- A temporary working folder for the *Dockerfile* and .NET Core example app. In this tutorial, the name *docker-working* is used as the working folder.
 
 ## Create .NET Core app
```

Comment:
Under "Your folder tree will look like the following:" it shows global.json, which I don't get, and I get an extra file named myapp.csproj.nuget.dgspec.json that isn't shown.